crates/task-impls/src/consensus.rs

```diff
@@ -124,11 +120,8 @@ pub struct ConsensusTaskState<
     /// last Timeout Certificate this node formed
     pub timeout_cert: Option<TimeoutCertificate<TYPES>>,
 
-    /// Global events stream to publish events
-    pub event_stream: ChannelStream<HotShotEvent<TYPES>>,
-
-    /// Event stream to publish events to the application layer
-    pub output_event_stream: ChannelStream<Event<TYPES>>,
+    /// Output events to application
+    pub output_event_stream: async_broadcast::Sender<Event<TYPES>>,
```

Comment:
Each task has an internal "event stream" (`Sender<T>` and `Receiver<T>`) defined in `Task` but that should be the channel that the tasks are communicating with each other.  The output event stream is different because it goes to the consumer of HotShot so tasks never have to consume events from other side of the channel.  It's set explicitly because it's not core to the behavior of the task, I guess.  And not every task has to output anything so it might not even be necessary have `output_event_stream` in every task